source/kraken/tests/realtime_test.cpp

```diff
@@ -1951,3 +1952,96 @@ BOOST_AUTO_TEST_CASE(skipped_stop_then_delay) {
     BOOST_REQUIRE_EQUAL(journeys.size(), 1);
     BOOST_CHECK_EQUAL(journeys[0].items.back().arrival, "20170101T083500"_dt);
 }
+
+/*delay: +5      0      0     +5      0
```

Comment:
nice ascii art!

I think it would be nice to add a small comment before though to have a bit more context (It's useless while reading the PR because we know the context, but with this 2k-lines file I think it would be worth it.

maybe `Test that we display delays on journeys only if the traveler is really impacted by it`